lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -57,6 +57,7 @@ class AwsCompileEventBridgeEvents {
           ],
         },
         schedule: { pattern: '^(?:cron|rate)\\(.+\\)$' },
+        enabled: { type: 'boolean' },
```

Comment:
It would be great to document this property in docs for `eventBridge` event

Reply:
Done